students/3/main.js

```diff
@@ -19,10 +19,24 @@ const ex3 = () => {
     console.log(shortestString(array));
 }
 
+const ex4 = () => {
+    let array = ["this", "is", "a", "test"];
+    console.log(countLetters(array));
+}
+
 //
 // Your functions here...
 //
 
+
+// EX 4
+const countLetters = (array) => {
+    let newArray = array.join ('');
+    newArray;
```

Comment:
Can we get rid of `newArray;?`  It appears to be hanging around, not providing much value.